### PR TITLE
Multiple values for custom javadoc tags

### PR DESCRIPTION
**Pull request by onewhl**
# Description

If there are several values for one tag, write them separately in the comment (`tag` `value`).

Fixes # (issue)

## Type of Change

- Minor bug fix (non-breaking small changes)

# How Has This Been Tested?

## Automated Testing

Specify tests that help to verify the change automatically.  

_Example:_ org.utbot.examples.algorithms.BinarySearchTest

## Manual Scenario 

Please, provide several scenarios that you went through to verify that the change worked as expected.  

# Checklist (remove irrelevant options):

_This is the author self-check list_

- [ ] The change followed the style guidelines of the UTBot project
- [x] Self-review of the code is passed
- [ ] The change contains enough commentaries, particularly in hard-to-understand areas
- [ ] New documentation is provided or existed one is altered
- [ ] No new warnings
- [ ] New tests have been added
- [ ] All tests pass locally with my changes

### CODE REVIEW

File: utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/javadoc/UtJavaDocInfoGenerator.kt [hunk 1]
```diff
@@ -51,22 +56,31 @@ class UtJavaDocInfoGenerator {
...
+            tags.mapIndexed { index, it ->
+                buildString {
+                    generateValue(this, it.dataElements)
+
+                    if (index < tags.size - 1) {
+                        this.append(", $BR_TAG")
+                    }
+                }
+            }
+                .forEach { builder.append(it) }
```
Comment (amandelpie) [inline utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/javadoc/UtJavaDocInfoGenerator.kt:80]: strange formatting, isn't it?

**Review by amandelpie — DISMISSED** [review]

**Comment by amandelpie** [comment]
@onewhl  could you please answer two of my minor comments?

**Comment by amandelpie** [comment]
Two tests related to the summary are failed in the build, could you please have a look?

File: utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/javadoc/UtJavaDocInfoGenerator.kt (hunk 1 above)
Comment (onewhl) [reply utbot-intellij/src/main/kotlin/org/utbot/intellij/plugin/javadoc/UtJavaDocInfoGenerator.kt:80]: fixed, thanks!

**Review by denis-fokin — APPROVED** [review]

**Review by Vassiliy-Kudryashov — APPROVED** [review]

**Pull request merged**